bindings/matrix-sdk-ffi/src/lib.rs

```diff
@@ -25,63 +24,6 @@ pub use matrix_sdk::ruma::{api::client::account::register, UserId};
 
 pub use self::{backward_stream::*, client::*, messages::*, room::*};
 
-pub fn guest_client(base_path: String, homeurl: String) -> anyhow::Result<Arc<Client>> {
```

Comment:
The guest client login is not used by element-x, only in a unit test of sample project that could be recreated in other ways, so I did not port this method yet. If I missed some other reason why this is important, I can bring it back